css/style.css

```diff
@@ -1,7 +1,297 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+}
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+body {
+	max-width: 1920px; /*резиновая ширина*/
+	min-width: 375px;
+	height: 100vh; /*значение высоты опр-ся по viewport*/
+	margin: 0 auto; /*выравнивание по центру*/
+	background: url(../img/iswanto-arif-448003-unsplash.png) no-repeat center; /*фоновая картинка*/
+	background-size: cover; /*картинка полностью покрывает блок при любых размерах*/
+	color: #E8EEE4; /*цвет шрифта*/
+	font: normal 22px/26px MuseoSansCyrl; /*стиль шрифта*/
+	overflow: hidden;
+}
+
+/*header styles start*/
+.container {
+	max-width: 1650px; /*резиновая ширина*/
+	margin: 0 auto; /*выравнивание по центру*/
+	padding: 0 15px;
+}
+
+.header__row { 
+	display: flex; 
+	align-items: flex-end; /*вертикальное выравнивание по нижнему краю контейнера*/
+	justify-content: space-between; /*выравнивание по ширине - по краям конейнера*/
+	width: 100%;
+	margin-left: 5%;
+	margin-bottom: 9.5%; /*отсутп вниз*/
+}
+
+.header__logo {
+	padding-top: 8.2%;
+	width: 146px;
+}
+
+.header__logo img {
+	max-width: 100%;
+	max-height: 100%;
+}
+
+.header__menu .menu{
+	display: flex;
+	padding-right: 30px;
+
+}
+
+.menu__link {
+	color: #E8EEE4;
+	margin-right: 33.75px; /*отступ справа у пуктов меню*/
+	text-decoration: none; /*убрали нижнее подчеркивание*/
+}
+
+.menu__link:hover, .menu__link:focus, .menu__link:active { /*стили для наведения на ссылки меню*/
+	opacity: .50;
+}
+
+.header__burger {
+	display: none;
+	width: 25px;
+	height: 17px;
+	position: relative;
+}
+
+.burger__line {
+	display: block;
+	width: 100%;
+	height: 3px;
+	background: #FCF5ED;
+	position: absolute;
+	left: 0;
+
+}
+
+.burger__line_first {
+	top: 0;
+}
+
+.burger__line_second {
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.burger__line_third {
+	bottom: 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455740)